<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>instOptionsDialog_q</class>
 <widget class="QDialog" name="instOptionsDialog_q">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Install options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="dialogTitleLine">
     <property name="text">
      <string>&lt;p align="center"&gt;&lt;b&gt;&lt;font size="+2"&gt;Install options&lt;/font&gt;&lt;/b&gt;&lt;/p&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="unameLbl">
       <property name="text">
        <string>User name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="uname"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="altAddressLabel">
       <property name="text">
        <string>Address to communicate with the firewall:</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="altAddress">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="verbose">
     <property name="text">
      <string>Verbose: print all commands as they are executed</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="quiet">
     <property name="text">
      <string>Quiet install: suppress output</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="copyFWB">
     <property name="text">
      <string>Store a copy of fwf file on the firewall</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="batchInstallText">
     <property name="text">
      <string>Check this option to install all remaining firewalls automatically using the same user name and parameters. This only works if the same credentials are used for all firewalls.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="batchInstall">
     <property name="text">
      <string>Perform batch install</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Install</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Skip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelAllButton">
       <property name="text">
        <string>Cancel All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>uname</tabstop>
  <tabstop>altAddress</tabstop>
  <tabstop>verbose</tabstop>
  <tabstop>quiet</tabstop>
  <tabstop>copyFWB</tabstop>
  <tabstop>batchInstall</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>cancelAllButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>instOptionsDialog_q</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>instOptionsDialog_q</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelAllButton</sender>
   <signal>clicked()</signal>
   <receiver>instOptionsDialog_q</receiver>
   <slot>cancelAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>batchInstall</sender>
   <signal>stateChanged(int)</signal>
   <receiver>instOptionsDialog_q</receiver>
   <slot>batchInstallStateChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cancelAll()</slot>
  <slot>batchInstallStateChange()</slot>
 </slots>
</ui>
